src/alert-manager/deploy/refresh.sh.template

```diff
@@ -17,5 +17,50 @@
 # DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+pushd $(dirname "$0") > /dev/null
```

Comment:
Following code sections for updating labels and pod running status are quite similar to the all services. Can we make a common bash template under paiLibrary/paiService and call it from here with service info as parameters?

Reply:
Have move it into the code of \pai\deployment\paiLibrary\paiService\service_management_refresh.py